charts/eks/values.yaml

```diff
@@ -238,11 +242,16 @@ api:
       memory: 300Mi
   priorityClassName: ""
   securityContext: {}
+  serviceAnnotations: {}
 
 # Core DNS settings
 coredns:
   image: public.ecr.aws/eks-distro/coredns/coredns:v1.8.7-eks-1-24-7
   replicas: 1
+  # CoreDNS service configurations
+  service:
+    # Extra Annotations
+    annotations: {}
```

Comment:
This has not been added to other charts. Let's keep them all in sync.

Reply:
This was already existing in other charts and was just missing in eks chart. So I added it here to keep all in sync.